packages/cosmic-swingset/Makefile

```diff
@@ -214,7 +213,7 @@ t1-provision-one-with-powers: wait-for-cosmos
 		  t1/$(BASE_PORT) $$addr $(AGORIC_POWERS) -ojson | tee /dev/stderr | grep -q '"code":0'; }
 
 # Send some USDC to the vbank/provision module account where it can be traded for IST.
-fund-provision-pool: wait-for-cosmos
```

Comment:
why not wait-for-cosmos?

Reply:
I'm trying to avoid waiting twice as long as `fund-acct` because the next line invokes a recursive make that already depends on `wait-for-cosmos`.